docs/source/models/vlm.rst

```diff
@@ -33,12 +33,7 @@ To initialize a VLM, the aforementioned arguments must be passed to the ``LLM``
 
 .. code-block:: python
 
-    llm = LLM(
-        model="llava-hf/llava-1.5-7b-hf",
-        image_token_id=32000,
-        image_input_shape="1,3,336,336",
-        image_feature_size=576,
-    )
+    llm = LLM(model="llava-hf/llava-1.5-7b-hf")
 
 .. important::
```

Comment:
This `important` section (and the same one below at line 107) needs to be updated - instead of saying `image_feature_size` needs to be specified, we can mention that currently we're conservative when it comes to profiling and this will be improved in the future.